dbuild/__init__.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import os
+import shutil
+from dbuild import exceptions
+
+from jinja2 import Environment, FileSystemLoader
+from docker import Client
+from tempfile import mkdtemp
+
+
+def docker_client(url='unix://var/run/docker.sock'):
+    """ return docker client """
+    return Client(url)
+
+
+def build_image(docker_client, path, tag, nocache=False):
+    """ Build docker image"""
+    lines = [line.values()[0] for line in docker_client.build(
+        path=path, rm=True, forcerm=True, tag=tag, decode=True, nocache=nocache)]
+    if any(isinstance(x, dict) for x in lines):
```

Comment:
This only happen once in a while when docker build failed. So I thought this would be better than doing below loop. If you see just using below loop is better, I can do that. Please let me know.